RevenueCatUI/CustomerCenter/Views/AppUpdateWarningView.swift

```diff
@@ -40,30 +40,47 @@ struct AppUpdateWarningView: View {
     }
 
     var body: some View {
-        List {
-            Section {
-                CompatibilityContentUnavailableView(
-                    localization[.updateWarningTitle],
-                    systemImage: "arrow.up.circle.fill",
-                    description: Text(localization[.updateWarningDescription])
-                )
+        Color(colorScheme == .light ? UIColor.secondarySystemBackground : UIColor.systemBackground)
+            .ignoresSafeArea()
```

Comment:
So I guess this works because we are adding the vertical padding right? Should we instead rely on the safe areas (and add any extra padding we need)?

Reply:
`ignoresSafeArea` is for the color to go all the way to the background

```swift
                    CompatibilityContentUnavailableView(
                        localization[.updateWarningTitle],
                        systemImage: "arrow.up.circle.fill",
                        description: Text(localization[.updateWarningDescription])
                    )
                    .padding(.horizontal, 16)
                    .padding(.vertical, 24)
```

The extra padding was to match more or less what we had before. The list has some built in spaces that are lost now because there's no list anymore

Reply:
Ahhh good point! Thanks for explaining 👍